.docker/images/govcms8/scripts/govcms-deploy

```diff
@@ -6,16 +6,50 @@
 # Ensure tmp folder always exists.
 mkdir -p /app/web/sites/default/files/private/tmp/
 
-# Run database updates if drupal is installed.
-if drush status --fields=bootstrap | grep -q "Successful"; then
-    mkdir -p /app/web/sites/default/files/private/backups/ && drush sql-dump --ordered-dump --gzip --result-file=/app/web/sites/default/files/private/backups/pre-deploy-dump.sql
-    drush en -y govcms_lagoon && drush pmu -y govcms_lagoon;
-    drush cim -y sync
-    if [ ! "${LAGOON_ENVIRONMENT_TYPE}" == "production" ]; then
+# Non production environments.
+if [[ "$LAGOON_ENVIRONMENT_TYPE" != "production" ]]; then
+    # Import production database on inital deployment.
+    if ! drush status --fields=bootstrap | grep -q "Successful"; then
+
+        # SQL sync only in Lagoon development environments.
+        if [[ ! -z "$LAGOON_ENVIRONMENT_TYPE" && "$LAGOON_ENVIRONMENT_TYPE" != "local" ]]; then
+          drush sql-sync @govcms-prod @self -y
+
+          # Base configuration import with development environment overrides.
+          drush cim -y sync
+          drush cim -y dev --partial
```

Comment:
Configuration should be imported - first the base/default config, and then the dev overrides.